File: BusinessLogicLayer/Models/LoginResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessLogicLayer.Models
{
    public class LoginResponse
    {
        public string Username { get; set; } = string.Empty;


        public string Role { get; set; } = string.Empty;

        public int? Id { get; set; }

        public string Token { get; set; } = string.Empty;
    }
}

File: UI-Angular/src/models/loginResponse.ts
export class LoginResponse{
    username: string;
    role: string
    id: number;
    token: string;

    constructor(username: string, role:string, id: number, token:string){
        this.username = username;
        this.role = role;
        this.id = id;
        this.token = token;
    }
}